pkg/process/net/common.go

```diff
@@ -226,6 +229,48 @@ func newSystemProbe(path string) *RemoteSysProbeUtil {
 	}
 }
 
+func (r *RemoteSysProbeUtil) DetectLanguage(pids []int32) ([]languagemodels.Language, error) {
+	procs := make([]*languagepb.Process, len(pids))
+	for i, pid := range pids {
+		procs[i] = &languagepb.Process{Pid: pid}
+	}
+	reqBytes, err := proto.Marshal(&languagepb.DetectLanguageRequest{Processes: procs})
+	if err != nil {
+		return nil, err
+	}
+
+	req, err := http.NewRequest(http.MethodGet, languageDetectionURL, bytes.NewBuffer(reqBytes))
```

Comment:
The windows unit tests check failures for `languageDetectionURL` look like legitimate issues, since this appears to be defined in a file built only on Linux.

Reply:
Ah thanks, that should be fixed now.